### DISCUSSION TITLE
Excek to PDF Page size issue

### DISCUSSION THREAD

**Discussion by Sarsewar** [ONLYOFFICE/DocumentServer#2688] [Q&A]
Please find attached [Demo_Excel.xlsx](https://github.com/exceljs/exceljs/files/14717537/Demo_Excel.xlsx), which we are currently using to convert to PDF. The conversion process is being conducted directly on DocuSign. However, we've encountered issues regarding page size and scaling, as illustrated in the screenshots below. Despite attempting conversion through other online free platforms specializing in direct XLSX to PDF conversion, we've encountered the same issues. Notably, the same Excel file converts correctly using Microsoft's "Save As" or "Print" functionality.

We have explored various combinations in the worksheet's XML, including adjustments to pageSize, scale, fitToWidth, and fitToHeight, yet none have proven successful. We seek your insights into the possible reasons for these challenges.

fitToPage | fitToWidth | fitToHeight | Scale
-- | -- | -- | --
0 | 0 | 0 | 0
0 | 0 | 0 | 1
0 | 0 | 1 | 0
0 | 0 | 1 | 1
0 | 1 | 1 | 1
0 | 1 | 0 | 1
0 | 1 | 1 | 0
0 | 1 | 0 | 0
1 | 0 | 0 | 0
1 | 0 | 0 | 1
1 | 0 | 1 | 0

![image](https://github.com/exceljs/exceljs/assets/26505197/1a890475-5121-4752-92a8-fabed22168b0)

Your guidance in resolving this matter is greatly appreciated.

**Comment by VladimirProkofiev52** [comment]
Hello! @Sarsewar 
I have reviewed your file and was able to reproduce 2 issues:  
1. I found that after to save the file in PDF format, the text located in the header of the resulting file is missing. 
2. After to save the file in PDF format, the table in the resulting file differs from its original appearance (the table formatting is lost). 

Unfortunately, I did not understand what you meant by issues related to page size and scaling.  

Please describe each problem in detail separately in the "Issues" section rather than in "Discussions," and we will do our best to assist you.  

Thank you for your report!

**Reply by Sarsewar** [reply]
Thanks @VladimirProkofiev52 

Please review the attached screenshot regarding the page size discrepancy in the original post. Note the differing sizes of the first and second pages. This issue is also tied to scaling. If we set the scale of both sheets to, for example, 30%, the page size of each sheet will be equal, but the content will appear very small.

**Reply by VladimirProkofiev52** [reply]
Please describe each issue in detail in the "Issues" section rather than in "Discussions."
We are ready to help resolve these problems, but we need a more detailed description of the scenario in which these issues arise so that we can make corrections and suggest solutions.
